pages/services/couchbase/1.0.0-6.0.0/troubleshooting/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Troubleshooting tips for Couchbase on DC/OS
+title: Troubleshooting
+menuWeight: 70
+model: /services/couchbase/data.yml
+render: mustache
+---
+
+#include /services/include/troubleshooting.tmpl
+
+## {{ model.techName }} diagnostic information
+
+More on {{ model.techName }} troubleshooting can be found [here](https://developer.couchbase.com/documentation/server/current/troubleshooting/troubleshooting-general-tips.html).
```

Comment:
Change to "More information about {{ model.techName }} troubleshooting ... "

Reply:
Changed.